test_e2e/python/test-ingest-delta-table-output.py

```diff
@@ -4,11 +4,20 @@
 
 @click.command()
 @click.option("--table-uri", type=str)
-def run_check(table_uri):
+@click.option("--aws-region", type=str)
+@click.option("--aws-access-key-id", type=str)
+@click.option("--aws-secret-access-key", type=str)
+def run_check(
+    table_uri: str, aws_region: str, aws_access_key_id: str, aws_secret_access_key: str
+) -> None:
     print(f"Checking contents of table at {table_uri}")
-    delta_table = DeltaTable(
-        table_uri=table_uri,
-    )
+    storage_options = {}
+    if aws_region and aws_access_key_id and aws_secret_access_key:
+        storage_options["AWS_REGION"] = aws_region
```

Comment:
Do we actually need to specify AWS region, since they're globally unique?

Reply:
I believe the `deltalake` library requires specifying the bucket region when reading Delta Tables from or writing them to S3. This can be tested with the following code:
```
from deltalake import DeltaTable

delta_table_path = "s3://path/to/delta/table/"
storage_options = {
    "AWS_ACCESS_KEY_ID": "AWS_ACCESS_KEY_ID",
    "AWS_SECRET_ACCESS_KEY": "AWS_SECRET_ACCESS_KEY",
    # "AWS_REGION": "us-east-2",
}
table = DeltaTable(delta_table_path, storage_options=storage_options)
```
 If the AWS_REGION is not provided then the following error is raised:
```
OSError: Generic S3 error: Received redirect without LOCATION, this normally indicates an incorrectly configured region
```

Reply:
I think bucket names have to be globally unique, but where AWS stores your data is region specific.